src/components/studies/participants/ParticipantManager.tsx

```diff
@@ -516,29 +520,62 @@ const ParticipantManager: FunctionComponent<ParticipantManagerProps> = () => {
   }
 
   const handleSearchParticipantRequest = async (searchedValue: string) => {
-    const result = await ParticipantService.getEnrollmentById(
-      study.identifier,
-      token!,
-      searchedValue,
-      tab
-    )
+    const promises = [
+      await ParticipantService.participantSearch(
+        study.identifier,
+        token!,
+        searchedValue,
+        tab,
+        'EXTERNAL_ID'
+      ),
+      await ParticipantService.participantSearch(
+        study.identifier,
+        token!,
+        searchedValue,
+        tab,
+        'PHONE_NUMBER'
+      ),
+    ]
+    const results = await Promise.all(promises)
+
+    let participants: ParticipantAccountSummary[] = []
+    let total = 0
+    const seenIds = new Set<string>()
+    for (const result of results) {
+      for (const participant of result.items) {
+        if (seenIds.has(participant.id)) continue
+        seenIds.add(participant.id)
+        total++
+        participants.push(participant)
+      }
+    }
 
-    if (result) {
-      const eventsMap: StringDictionary<ParticipantRelevantEvents> =
-        await ParticipantService.getRelevantEventsForParticipants(
+    if (total > 0) {
+      const participantData = []
+      const eventsMapPromises = participants.map(async participant => {
```

Comment:
we already have functionality in getParticipants where we join participants with relevant events. Let's instead refactor this as a function that we can also use here